app/uk/gov/hmrc/agentsexternalstubsfrontend/controllers/UserController.scala

```diff
@@ -326,7 +328,10 @@ object UserController {
       "recordIds"         -> ignored[Option[Seq[String]]](None),
       "address"           -> optional(addressMapping),
       "strideRoles" -> optional(nonEmptyText)
-        .transform[Seq[String]](_.map(_.split(",").toSeq).getOrElse(Seq.empty), s => Some(s.mkString(",")))
+        .transform[Seq[String]](_.map(_.split(",").toSeq).getOrElse(Seq.empty), s => Some(s.mkString(","))),
+      "suspendedServices" -> optional(seq(optional(nonEmptyText))).transform[Option[Set[String]]](_.map(_.collect {
```

Comment:
do we need two options ? cant we simply have optional(seq(nonEmptyText)) ?

Reply:
the whole thing is optional and each individual field is optional. I did have optional(seq(nonEmptyText)) before and was getting a field required validation error!

Reply:
ok,  but isn't this same as `principalEnrolments`  or ` principalEnrolments` above  which are `optional(seq(enrolmentMapping))` ?

Reply:
yeah but enrolmentsMapping is also an option so that is also optional(seq(optional...